src/RFC-0122_Burning.md

```diff
@@ -181,6 +202,46 @@ $$
 
 When verifying total chain emission, equation (5) MUST hold.
 
+
+### Option 2 for Proof of burn
+
+If we don't want to look into the output MMR, we can include a second MMR root and size counter to the header to track the commitment. 
+
+```rust,ignore
+pub struct BlockHeader {
+    /// Version of the block
+    pub version: u16,
+    ...
+    /// Sum of kernel offsets for all kernels in this block.
+    pub total_kernel_offset: BlindingFactor,
+    /// Sum of script offsets for all kernels in this block.
+    pub total_script_offset: BlindingFactor,
+    /// Sum of all burned output commitments in this block.
+    pub burned_total: Commitment,
+    /// Merkle root of the burned MMR tree at this block height
+    pub burned_mr;
+    /// Size of the burned MMR tree at this height. 
+    pub burned_mmr_size;
+    /// Nonce increment used to mine this block.
+    pub nonce: u64,
+    /// Proof of work summary
+    pub pow: ProofOfWork,
+}
+```
+
+[burned_mr]: #burned_mr "Burned Merkle root"
+
+This is the Merkle root of the burned outputs.
+
+The kernel_mr MUST conform to the following:
+
+* Must be transmitted as an array of unsigned 8-bit integers (bytes) in little-endian format.
```

Comment:
Rather add this info to the other relevant hashing RFC. 

It should also be hashed as a VarInt. 

Transfer doesn't really matter, since it is sent via Protobuf

Reply:
We should probably update the other consensus rfc then for the other hashes.